SQLite.CodeFirst/Internal/Builder/CreateTableStatementBuilder.cs

```diff
@@ -19,13 +21,19 @@ public CreateTableStatementBuilder(EntitySet entitySet, AssociationTypeContainer
 
         public CreateTableStatement BuildStatement()
         {
+            var nonAutoincrementKeys = entitySet.ElementType.KeyMembers
+                .Where(km => (km as EdmProperty)?.GetCustomAnnotation<AutoincrementAttribute>() == null);
```

Comment:
I found that it may not be possible to use my approach.
So its necessary to specify it as a constraint but it can also be kind of duplicated.
That means the logic can't be moved like I suggested.

```
CREATE TABLE TeamPlayer1 (
    **Id        INTEGER PRIMARY KEY AUTOINCREMENT,**
    Number    INT            NOT NULL,
    TeamId    INT            NOT NULL,
    FirstName NVARCHAR (50)  COLLATE NOCASE,
    LastName  NVARCHAR (50),
    Street    NVARCHAR (100),
    City      NVARCHAR       NOT NULL,
    Mentor_Id INT,
    **PRIMARY KEY (
        Id
    ),**
    FOREIGN KEY (
        Mentor_Id
    )
    REFERENCES TeamPlayer (Id),
    FOREIGN KEY (
        TeamId
    )
    REFERENCES MyTable (Id) ON DELETE CASCADE
);
```

But I'll need to have a deeper look.
I tried to add the Autoincrement attribute to the Id property of the Team class (see console project) and it does not work as the Column type will be an "int" instead of an "INTEGER".